apps/web/app/(auth-pages)/sign-in/page.tsx

```
@@ -1,146 +1,171 @@
+    if (!isEmailInvalid) handleAuth();
+    resetValidation();
+  };
 
-	useEffect(() => {
-		if (authSuccess) {
-			// If the user is authenticated, we can use the stellarUserAddress later
-			console.log('stellarUserAddress', stellarUserAddress)
-			router.push('/')
-		}
-	}, [authSuccess, router, stellarUserAddress])
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: stellarUserAddress

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)